components/Layout/Navbar.tsx

```diff
@@ -9,72 +9,165 @@ import {
   MenuButton,
   MenuItem,
   MenuList,
+  Stack,
+  Text,
+  useDisclosure,
 } from '@chakra-ui/react'
+import { useUser } from 'components'
+import AuthModal from 'components/Auth/AuthModal'
+import { AvatarIcon } from 'components/Icons'
 
 export const Navbar = (): JSX.Element => {
+  const { onOpen, isOpen, onClose } = useDisclosure()
+  const { loading, userData } = useUser()
+
   return (
-    <Flex
-      alignItems="center"
-      bgColor="#204045"
-      color="white"
-      p="20px"
-      justify="space-between"
+    <Stack
+      w="100%"
+      alignItems={'center'}
+      color="#667085"
+      borderBottom="1px solid #F2F4F7"
+      bgColor="white"
     >
-      <Link href="/">
-        <Image
-          src="/manyc_logo_no_background.png"
-          w="108px"
-          h="96px"
-          alt="Mutual Aid NYC"
-        />
-      </Link>
-      <HStack
-        spacing={{ base: '16px', lg: '32px' }}
+      <AuthModal
+        isOpen={isOpen}
+        onClose={onClose}
+        message={'Login to MANY Resources Hub'}
+      />
+      <Flex
+        alignItems="center"
+        justify="space-between"
+        w="100%"
+        maxW="7xl"
         display={{ base: 'none', lg: 'inherit' }}
       >
-        <Link href="/">Resource lists</Link>
-        <Link href="https://resources.mutualaid.nyc/">Resource library</Link>
-        <Link href="/create-list">Create new list</Link>
-        <Link href="https://mutualaid.nyc/submit-a-resource/">
-          Submit a resource
-        </Link>
-        <Link href="https://mutualaid.nyc/">About Mutual Aid NYC</Link>
-      </HStack>
-      <Menu>
-        <MenuButton
-          display={{ base: 'inherit', lg: 'none' }}
-          as={IconButton}
-          aria-label="Navbar menu"
-          icon={<HamburgerIcon />}
-          bgColor="#204045"
-        />
-        <MenuList
-          display={{ base: 'inherit', lg: 'none' }}
-          bgColor="darkPurple"
-        >
-          <MenuItem
-            _focus={{ background: 'darkPurple', filter: 'brightness(150%)' }}
+        <HStack spacing={{ base: '16px', lg: '32px' }}>
+          <Link href="/">
+            <Image
+              src="/new_manyc_logo.png"
```

Comment:
Instead of calling this the `new` logo, just delete the old one + name this one by the color it is because [in the Figma](https://www.figma.com/file/weR9JkmoDTVtQMuVeBViXa/Designs?t=p19dwMOcnAHeZ1Vd-0), there are multiple colors of this new logo.